### PR TITLE
Add null host name check in WSMan

### LINKED ISSUE #11278: WSMan provider throws exception for non-existent path with -PathType Container
# Steps to reproduce

```powershell
test-path wsman:\foo -pathtype container
```

# Expected behavior

The command returns `$false`.

# Actual behavior

Exception is thrown: "Test-Path: Value cannot be null. (Parameter 'value')":

image

The `$Error[1].Exception.InnerException.InnerException.StackTrace` (since `$Error[0]` is the error for `Invoke`) shows:
```
   at System.String.StartsWith(String value, StringComparison comparisonType)
   at Microsoft.WSMan.Management.WSManConfigProvider.NormalizePath(String path, String host)
   at Microsoft.WSMan.Management.WSManConfigProvider.IsItemContainer(String path)
   at System.Management.Automation.SessionStateInternal.IsItemContainer(CmdletProvider providerInstance, String path, CmdletProviderContext context)
```

# Environment data

```none
Name                           Value
----                           -----
PSVersion                      7.0.0-preview.6
PSEdition                      Core
GitCommitId                    7.0.0-preview.6
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0, ...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### PR DESCRIPTION
**Pull request by iSazonov**
# PR Summary

Add null check for host name before path normalization.

## PR Context

Fix #11278

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: tools/releaseBuild/setReleaseTag.ps1 [hunk 1]
```diff
@@ -36,6 +36,7 @@ function New-BuildInfoJson {
     ## Get the UTC time and round up to the second.
     $dateTime = [datetime]::UtcNow
     $dateTime = [datetime]::new($dateTime.Ticks - ($dateTime.Ticks % [timespan]::TicksPerSecond), $dateTime.Kind)
+    $dateTime = $dateTime.ToString("o")
```
Comment (SteveL-MSFT) [inline tools/releaseBuild/setReleaseTag.ps1]: Why is this change part of this PR?

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: tools/releaseBuild/setReleaseTag.ps1 (hunk 1 above)
Comment (iSazonov) [reply tools/releaseBuild/setReleaseTag.ps1]: :-) I wonder too. Removed.

**Review by PaulHigin — APPROVED** [review]
LGTM

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by anmenaga** [comment]
CodeFactor issues are false positives.

**Pull request merged**